pkg/gameplay/game.go

```diff
@@ -45,8 +46,22 @@ type GameStore interface {
 	SetGameEventChan(c chan<- *entity.EventWrapper)
 }
 
+type DummyListStatStore struct {
+}
+
+func (lss DummyListStatStore) AddListItem(gameId string, playerId string, statType int, time int64, item interface{}) error {
+	// Do nothing for now
+	return nil
+}
+
+func (lss DummyListStatStore) GetListItems(statType int, gameIds []string, playerId string) ([]*entity.ListItem, error) {
+	return []*entity.ListItem{}, nil
+}
+
 type ConfigCtxKey string
 
+var listStatStore = DummyListStatStore{}
+
```

Comment:
I don't think this or the DummyListStatStore need to be defined here. Instead some function here (perhaps PlayMove) would take in a ListStatStore interface. The current signature looks like this for example:

```
func PlayMove(ctx context.Context, gameStore GameStore, userStore user.Store, userID string,
	cge *pb.ClientGameplayEvent) error {
```

Note that the GameStore and user.Store types are interfaces and not necessarily structs. Additionally, PlayMove perhaps could take in a ListStatStore interface. It would have to pass it down to some other functions. Then we could just define a "dummy" list stat store at the very top level.